test/hotspot/jtreg/runtime/valhalla/inlinetypes/ValueTearing.java

```diff
@@ -43,7 +43,7 @@
 import static jdk.test.lib.Asserts.*;
 
 /*
- * @test ValueTearing
+ * @test
  * @ignore
```

Comment:
Just checking: Is there a tracking issue to re-enable this test?

Reply:
I assume it's related to [JDK-8293983](https://bugs.openjdk.org/browse/JDK-8293983).